src/esp/gfx/Renderer.cpp

```diff
@@ -79,7 +87,37 @@ struct Renderer::Impl {
                 esp::gfx::Renderer::Impl::RendererShaderType::
                     DepthTextureVisualizer);
 
+#ifdef ENABLE_VISUALIZATION_WORKAROUND_ON_MAC
+        depthBufferImage_ = tgt.getDepthTexture().image(
+            0, {Mn::GL::PixelFormat::DepthComponent, Mn::GL::PixelType::Float},
+            Mn::GL::BufferUsage::StaticRead);
+
+        // This takes the above output image (which is depth) and "reinterprets"
+        // it as R32F. In other words, the image below serves as an "image
+        // view".
+        Mn::GL::BufferImage2D clonedDepthImage{
+            depthBufferImage_->storage(), Mn::PixelFormat::R32F,
+            depthBufferImage_->size(),
+            Mn::GL::Buffer::wrap(depthBufferImage_->buffer().id(),
```

Comment:
FYI @mosra: the latest optimization attempt did not give us further performance gain. :)

Reply:
Yeah, not like that, you're still allocating a new image in every call to `image()` above. It has to be instead

```cpp
        if(!depthBufferImage_) // create a BufferImage instance, if not already
          depthBufferImage_.emplace(Mn::GL::PixelFormat::DepthComponent, Mn::GL::PixelType::Float);
        tgt.getDepthTexture().image(0, *depthBufferImage_, Mn::GL::BufferUsage::StaticRead);
```

Reply:
OK. Thanks. Changed.
Haha, maybe 1 FPS improvement. 
<img width="796" alt="Screen Shot 2021-04-28 at 11 15 05 AM" src="https://user-images.githubusercontent.com/931093/116453021-38757980-a813-11eb-9a29-71ef830add67.png">

Reply:
Heh, okay. Well, who cares, it's just MacGL anyway :)